---
layout: post
title: "DropLasso: A robust variant of Lasso for single cell RNA-seq data"
date: 2018-02-26 15:10:44
categories: arXiv_CV
tags: arXiv_CV Sparse
author: Beyrem Khalfaoui (CBIO), Jean-Philippe Vert (CBIO, DMA)
mathjax: true
---

* content
{:toc}

##### Abstract
Single-cell RNA sequencing (scRNA-seq) is a fast growing approach to measure the genome-wide transcriptome of many individual cells in parallel, but results in noisy data with many dropout events. Existing methods to learn molecular signatures from bulk transcriptomic data may therefore not be adapted to scRNA-seq data, in order to automatically classify individual cells into predefined classes. We propose a new method called DropLasso to learn a molecular signature from scRNA-seq data. DropLasso extends the dropout regularisation technique, popular in neural network training, to esti- mate sparse linear models. It is well adapted to data corrupted by dropout noise, such as scRNA-seq data, and we clarify how it relates to elastic net regularisation. We provide promising results on simulated and real scRNA-seq data, suggesting that DropLasso may be better adapted than standard regularisa- tions to infer molecular signatures from scRNA-seq data.

##### Abstract (translated by Google)
单细胞RNA测序（scRNA-seq）是一种快速增长的方法，用于同时测量许多单个细胞的全基因组转录组，但会导致数据嘈杂且伴随多种丢失事件。为了自动将单个细胞分类到预定义的类别，因此从大量转录组学数据中学习分子特征的现有方法可能不适用于scRNA-seq数据。我们提出一种名为DropLasso的新方法来从scRNA-seq数据中学习分子签名。 DropLasso扩展了神经网络训练中流行的丢失正则化技术，以估计稀疏线性模型。它非常适合于数据丢失的数据，如scRNA-seq数据，并且我们阐明了它与弹性净正则化之间的关系。我们对模拟和真实的scRNA-seq数据提供了有希望的结果，表明DropLasso可能比标准调整更好地适用于从scRNA-seq数据推断分子特征。

##### URL
[http://arxiv.org/abs/1802.09381](http://arxiv.org/abs/1802.09381)

##### PDF
[http://arxiv.org/pdf/1802.09381](http://arxiv.org/pdf/1802.09381)

