---
layout: post
title: "BioSEAL: In-Memory Biological Sequence Alignment Accelerator for Large-Scale Genomic Data"
date: 2019-01-17 10:13:22
categories: arXiv_CV
tags: arXiv_CV
author: Roman Kaplan, Leonid Yavits, Ran Ginosar
mathjax: true
---

* content
{:toc}

##### Abstract
Genome sequences contain hundreds of millions of DNA base pairs. Finding the degree of similarity between two genomes requires executing a compute-intensive dynamic programming algorithm, such as Smith-Waterman. Traditional von Neumann architectures have limited parallelism and cannot provide an efficient solution for large-scale genomic data. Approximate heuristic methods (e.g. BLAST) are commonly used. However, they are suboptimal and still compute-intensive. In this work, we present BioSEAL, a Biological SEquence ALignment accelerator. BioSEAL is a massively parallel non-von Neumann processing-in-memory architecture for large-scale DNA and protein sequence alignment. BioSEAL is based on resistive content addressable memory, capable of energy-efficient and high-performance associative processing. We present an associative processing algorithm for entire database sequence alignment on BioSEAL and compare its performance and power consumption with state-of-art solutions. We show that BioSEAL can achieve up to 57x speedup and 156x better energy efficiency, compared with existing solutions for genome sequence alignment and protein sequence database search.

##### Abstract (translated by Google)


##### URL
[https://arxiv.org/abs/1901.05959](https://arxiv.org/abs/1901.05959)

##### PDF
[https://arxiv.org/pdf/1901.05959](https://arxiv.org/pdf/1901.05959)

